.github/workflows/update-jekyll-e2e-tests-results.yml

```diff
@@ -0,0 +1,47 @@
+name: Update Jekyll Page with Test Report
+
+on:
+  workflow_run:
+    workflows: ['Meshery UI and Server']  
+    types: [completed]
+
+permissions:
+  actions: read
+  contents: write
+
+jobs:
+  update-jekyll:
+    if: >
+        github.repository == 'meshery/meshery' && 
+        github.event.workflow_run.event == 'push' && 
+        github.event.workflow_run.head_branch == 'master'
```

Comment:
This condition will work but may be a bit heavy-handed. New to github actions and i believe it can simplify would appreciate some input w/ those more familiar.